shell/platform/tizen/accessibility_settings.cc

```diff
@@ -9,48 +9,49 @@
 
 // SYSTEM_SETTINGS_KEY_MENU_SYSTEM_ACCESSIBILITY_HIGHCONTRAST = 10059 has been
 // defined in system_settings_keys.h only for TV profile.
-#define SYSTEM_SETTINGS_KEY_MENU_SYSTEM_ACCESSIBILITY_HIGHCONTRAST 10059
+#define SYSTEM_SETTINGS_KEY_ACCESSIBILITY_HIGHCONTRAST \
+  system_settings_key_e(10059)
 
 namespace flutter {
 
 AccessibilitySettings::AccessibilitySettings(FlutterTizenEngine* engine)
     : engine_(engine) {
 #ifndef WEARABLE_PROFILE
   bool tts_enabled = false;
-  int result = system_settings_get_value_bool(
+  int ret = system_settings_get_value_bool(
       SYSTEM_SETTINGS_KEY_ACCESSIBILITY_TTS, &tts_enabled);
-  if (result == SYSTEM_SETTINGS_ERROR_NONE) {
-    if (tts_enabled) {
-      screen_reader_enabled_ = tts_enabled;
-      engine_->SetSemanticsEnabled(tts_enabled);
-    }
-  } else {
+  if (ret != SYSTEM_SETTINGS_ERROR_NONE) {
     FT_LOG(Error) << "Failed to get value of accessibility tts.";
   }
-
-  // Add listener for accessibility tts
+  if (tts_enabled) {
+    screen_reader_enabled_ = tts_enabled;
+    engine_->SetSemanticsEnabled(tts_enabled);
+  }
   system_settings_set_changed_cb(SYSTEM_SETTINGS_KEY_ACCESSIBILITY_TTS,
                                  OnScreenReaderStateChanged, this);
 #endif
 
 #ifdef TV_PROFILE
```

Comment:
Is negative color and high contrast properties for TV only?

Reply:
Yes, both are only for TV as stated in https://github.com/flutter-tizen/flutter-tizen/issues/151#issuecomment-1064857205.

- High contrast: The `SYSTEM_SETTINGS_KEY_MENU_SYSTEM_ACCESSIBILITY_HIGHCONTRAST` flag is only defined for TV.
- Negative color: The `system_settings_get_value_bool` call (with `SYSTEM_SETTINGS_KEY_ACCESSIBILITY_NEGATIVE_COLOR` as argument) fails on non-TV devices.

Reply:
Wow, I have one question, where did you get the Accessibility support status table?

Reply:
That's what I created. The table shows our current accessibility support status and https://github.com/flutter-tizen/flutter-tizen/wiki/Limitations#apis has a link to the comment.

Reply:
Thank you for quick response :)